spec/requests/api/visitor_can_see_details_of_an_article_spec.rb

```diff
@@ -22,5 +22,9 @@
     it 'should return content articles' do
       expect(response_json['article']['content']).to eq 'Article content will go here for the user to read.'
     end
+
+    it 'should return article category' do
+      expect(response_json['article']['category']).to eq 'latest_news'
```

Comment:
Another detail. In the session we had this morning (Saturday), when we ware returning a single article, you guys said it came through wrapped in a `articles` key. According to this, it comes in as an `article`. Which is it?

Reply:
We will modify the front end in order to expect `article` instead of `articles`.